jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -69,17 +70,23 @@ pipeline {
 
         stage('Process Manifest Images') {
             steps {
-                gitCommitWhenChanges("Updated image_mirror_mapping") { msgs ->
-                    echo "Process Manifest Images"
+                gitCommitWhenChanges("Updated images") { msgs ->
```

Comment:
I've updated this because the commit itself doesn't contain any image image_mirror_mapping files. We don't commit those.

Reply:
I don't mind the renaming, but unsure of the reasoning. If there were no changes in image_mirror_mapping or one wasn't added, then i expect there to be no changes to commit. Do we not just need to add the `sh "git add ."` line here to make sure the new image_mirror_mapping file is committed?

Reply:
To me the message `Updated image_mirror_mapping` would indicate that some updates were made to those files. We don't commit those files, what we do in this commit is update the images in csv. An example of the commit is here: https://github.com/integr8ly/integreatly-operator/commit/bb2b2385b83d271b1e3c4150867af87c4b7fc47d That commit only contains updates to images in the csv.yaml

Reply:
Yes, but the image mirror mapping is supposed to be getting committed is what I'm saying, we need to add this line `sh "git add ."`. Can you add it in this PR?

Reply:
Aha. I thought we shouldn't commit those. Sure will add that now.

Reply:
We want to keep everything on the branch that is required to make it work, including the mirroring file. When the product is eventually GA'd and we no longer need this mirroring, we will delete the file and set the registry back to the correct ones. We have this ticket to create some checks that will fail the PR tests if this file exists https://issues.redhat.com/browse/DEL-53